Mathlib/Tactic/Common.lean

```diff
@@ -11,6 +11,11 @@ import Qq
 -- Tools for analysing imports, like `#find_home`, `#minimize_imports`, ...
 import ImportGraph.Imports
 
+-- Currently we don't need to import all of ProofWidgets,
+-- but without this, if you don't run `lake build ProofWidgets` then `make test` will fail.
+-- Hopefully `lake` will be able to handle tests later.
+import ProofWidgets
```

Comment:
This adds an MIT-licensed dependency to Mathlib, which is otherwise Apache-licensed; can you use a more fine-grained import?

Reply:
Perhaps more pragmatically; do we really want to add dependencies to all mathlib users to save one line in a CI script?

Reply:
What's the legal theory here?

Reply:
Maybe I'm imagining the legal issue, certainly I misremembered the relative strength of the licenses.

Reply:
I'm no lawyer so don't take what I'm saying very seriously, but MIT is extremely permissive, and it says it only requires that the license notice be included with the copy. It's even preserved when cloned by `lake` as `.lake/packages/proofwidgets/LICENSE`, in case that matters.

Reply:
I'm pretty sure that license file is apache, and the mit license is embedded within one of the build artifacts

Reply:
It's not just to save one line in a CI script.

The real problem here is that
```
lake exe cache get
lake build
make test
```
currently fails in a fresh Mathlib. Someone needs to run `lake build ProofWidgets`.

Finding the smallest import that suffices for `make test` would be fine.

Reply:
Does replacing the middle line with `lake build Mathlib ProofWidgets` work?

Reply:
Yes, but that that's still a terrible user experience to need to know that!

Reply:
@eric-wieser, did you want to minimize imports? If not I'll merge as is?

Reply:
I'm curious what @digama0's thoughts are here (about adding the dependency, I think my license concerns are moot).

Is there a reason that adding `lake build` into `make test` is a bad option?

Reply:
Adding at least `lake build ProofWidgets` seems sound. As for adding the dependency, `shake` is not in the business of prescribing Mathlib's policy toward intentionally broad imports, that's why `noshake.json` exists (esp. the `ignoreImport` list). I have no strong opinions regarding whether `import Mathlib` should imply `import ProofWidgets`, but I agree that we should make the tests work without this additional step, and the right place to put `import ProofWidgets` is in the `make test` target. (The actual right place is in `lake` but until `lake test` comes about we have to do it the hard way.)